improves wording (#308)
rsocket/rsocket: Extensions/PerStreamDataMimeTypesDefinition.md

_This extension specification is currently incubating.  While incubating the version is 0._

## Introduction

> The definition of the `Payload` data MIME type is an integral part of the RSocket and Extension specifications. However, due to its definition as per connection, it is non-trivial to make it suitable for the cases where each stream can have different MIME types within a single connection. Therefore, this extension specification provides an interoperable structure for metadata payloads to contain data MIME types' information. It is designed such that Requester can define using [CompositeMetadata][cm] expected request `Payload` [data MIME type][dmt] and a set of [accepted][admt] for the response `Payload`(s) data MIME types. If the data MIME type is not declared, then the MIME Type defined by `ConnectionSetupPayload` MUST be used instead. As per definition, the declaration of accepted data MIME types is on the Requester and may be considered to use by the Responder in order to encode the response data. If the response `Payload`(s) data MIME type is other than specified by the Requester, then the Responder MUST specify the response [data MIME type][dmt] in the Metadata Payload using [CompositeMetadata][cm]. Declaration of the [accepted data MIME types][admt] by the Responder MUST be considered invalid and ignored by the Requester.

Reviewer: ```suggestion
The definition of the `Payload` data MIME type is an integral part of the RSocket and Extension specifications. However, due to its definition as per connection, it is non-trivial to make it suitable for cases where streams within a single connection need to use different MIME types. Therefore, this extension specification provides an interoperable structure for metadata payloads to declare MIME type information. It is designed such that Requester can use [CompositeMetadata][cm] to declare the request `Payload` [data MIME type][dmt] and a set of expected, [acceptable][admt] response `Payload`(s) data MIME types. If data MIME types are declared, then the MIME type declared in the `ConnectionSetupPayload` MUST be used instead. As per definition, the declaration of accepted data MIME types is made by the Requester and may be considered by the Responder for encoding response data. If the response `Payload`(s) data MIME type is other than specified by the Requester, then the Responder MUST specify the response [data MIME type][dmt] in the Metadata Payload using [CompositeMetadata][cm]. Declaration of the [accepted data MIME types][admt] by the Responder MUST be considered invalid and ignored by the Requester.
```

Author: applied
